src/pymodaq/examples/parameter_ex.py

```diff
@@ -67,8 +67,14 @@ class ParameterEx:
              'tip': 'Press Ctrl+click  to select items in any order'},
             {'title': 'Selectable items', 'name': 'itemsbis', 'type': 'itemselect',
              'value': dict(all_items=['item1', 'item2', 'item3'], selected=['item2']),
-             'tip': 'If show_pb is True, user can add items to the list', 'show_pb': True},
-        ]},
+             'tip': 'If show_pb is True, user can add items to the list', 'show_pb': True,},
+            {'title': 'Checkable items', 'name': 'itemsbisbis', 'type': 'itemselect',
+             'value': dict(all_items=['item1', 'item2', 'item3'], selected=['item2']),
+             'tip': 'If checkbox is True, user can select item by checking/unchecking items', 'show_pb': True,'checkbox':True,},                        
+            {'title': 'Dragable items', 'name': 'itemsbisbisbis', 'type': 'itemselect',
+             'value': dict(all_items=['item1', 'item2', 'item3'], selected=['item2']),
+             'tip': 'If dragdrop is True, user can drag or drop items inside the list', 'show_pb': True,'checkbox':True,'dragdrop':True}, 
```

Comment:
is there a use in allowing drag/drop?

Reply:
I would again argue that it makes the item more general.
drag/drop cannot really hurt as long as it is internal and is used in a simple qlistwidget

Reply:
With the checkable items, it allows you to change the order of actuators that you desire

Reply:
with the checkable items, the order of the items where you clicked is not setting the order?